src/main/kotlin/io/easybreezy/project/application/issue/command/parser/PriorityConverter.kt

```diff
@@ -0,0 +1,23 @@
+package io.easybreezy.project.application.issue.command.parser
+
+import com.google.inject.Inject
+import io.easybreezy.infrastructure.query.QueryExecutor
+import io.easybreezy.project.application.project.queryobject.LastHighestPriorityQO
+import io.easybreezy.project.application.project.queryobject.LastLowestPriorityQO
+import io.easybreezy.project.model.issue.Priority
+import java.util.UUID
+
+class PriorityConverter @Inject constructor(
+    private val queryExecutor: QueryExecutor
+) {
+    suspend fun convert(project: UUID, title: String?): Priority {
+        return when (title?.toLowerCase()) {
+            "high" -> Priority.high()
```

Comment:
We should add the ability to use many languages. May create a file with map x keywords and translation